book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals

```

Comment:
- [ ] Please set the `rentals` variable as an attribute accessor. The reason is that we will want to read and set the rentals. Currently, your implementation is for reading.

Reply:
This has been noted and the changes have been made